lib/sgtn-client/api/source.rb

```diff
@@ -40,6 +40,10 @@ def self.getSources(component, locale)
 
       def self.loadBundles(locale)
         SgtnClient.logger.debug "[Source][loadBundles]locale=#{locale}"
+        if locale.nil?
```

Comment:
Do not hide the exception here as nil calue for locale is invalid for this API method.